### ISSUE TITLE
How to cite the Documenter.jl package

### ISSUE THREAD

**Issue by jschepers** [JuliaDocs/Documenter.jl#2386]
Hi,

we are using Documenter.jl in our packages and would like to cite it in our work. However, I couldn't find a citation file in the repo or a Zenodo link or something similar. Maybe I overlooked something. How should the package be cited?

Best,
Judith

**Comment by goerz** [comment]
You should probably wait for an official answer from @mortenpi as the primary maintainer, but If I were to cite Documenter.jl or any other Github package that does not have an associated paper, I would do it with

> M. Piibeleht and Contributors. "Documenter: A documentation generator for Julia". code: https://github.com/JuliaDocs/Documenter.jl

**Comment by jschepers** [comment]
Okay, thank you for your response.

**Comment by mortenpi** [comment]
Minimally, @MichaelHatherly and @fredrikekre should also be credited. I am not quite sure what the best practice is with open source software with many contributors. But one suggestion would be to just follow what we have in our license file:

https://github.com/JuliaDocs/Documenter.jl/blob/223102fa86558652d2819f13687c7839614c2546/LICENSE.md?plain=1#L3

The title

> Documenter: A documentation generator for Julia

seems fine to me, with a link to the GitHub repository, like @goerz suggested.

**Comment by MichaelHatherly** [comment]
> But one suggestion would be to just follow what we have in our license file:

Yeah, just matching the license seems like the most common thing to do here.

**Comment by jschepers** [comment]
> Minimally, @MichaelHatherly and @fredrikekre should also be credited. I am not quite sure what the best practice is with open source software with many contributors. But one suggestion would be to just follow what we have in our license file:
> 
> https://github.com/JuliaDocs/Documenter.jl/blob/223102fa86558652d2819f13687c7839614c2546/LICENSE.md?plain=1#L3
> 
> The title
> 
> > Documenter: A documentation generator for Julia
> 
> seems fine to me, with a link to the GitHub repository, like @goerz suggested.
> 
> So, altogether, like this probably then:
> 
> > M. Hatherly, M. Piibeleht, F. Ekre, and Contributors. "Documenter: A documentation generator for Julia". code: https://github.com/JuliaDocs/Documenter.jl

Okay, thank you for the additional response. Then I will do it like this.
